apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -1,19 +1,11 @@
 <div
   class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
   [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px" />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px" />
-
   <section>
+    <ng-content />
     <app-list-item
       *ngFor="let item of list"
-      [name]="item.firstname"
+      [name]="item.firstname ?? item.name"
       [id]="item.id"
       [type]="type">
     </app-list-item>
```

Comment:
What happen if I want to add an icon only for city card. To make it more customizable, look at `ngTemplateOutlet`